windows/stat_hosttime/stat_hosttime.yml

```diff
@@ -0,0 +1,73 @@
+#
+- name: stat_hosttime
+  hosts: localhost
+  gather_facts: false
+  vars_files:
+    - "{{ testing_vars_file | default('../../vars/test.yml') }}"
+  tasks:
+    - name: "Test case block"
+      block:
+        - name: "Skip test case if no VMware tools"
+          include_tasks: ../setup/test_setup.yml
+          vars:
+            skip_test_no_vmtools: true
+
+        - name: "Set time zone of guet OS to UTC"
+          include_tasks: ../utils/win_execute_cmd.yml
+          vars:
+            win_powershell_cmd: "Set-TimeZone -Id UTC"
+
+        - name: "Get host time by VMware tools in guet OS"
+          ansible.windows.win_shell: "& 'C:\\Program Files\\VMware\\VMware Tools\\VMwareToolboxCmd.exe' stat hosttime"
+          register: stat_time_output
+          delegate_to: "{{ vm_guest_ip }}"
+          ignore_unreachable: true
+          poll: 0
+
+        # At the same time, get ESXi host time by Ansible module community.vmware.vmware_host_facts
+        - name: "Get real host time on {{ esxi_hostname }}"
+          include_tasks: ../../common/esxi_get_host_facts.yml
+
+        - name: "Set the fact of host time got from {{ esxi_hostname }}"
+          ansible.builtin.set_fact:
+            real_time: "{{ ' '.join((host_facts.ansible_facts.host_date_time.date, host_facts.ansible_facts.host_date_time.time)) }}"
+
+        - name: "Set the fact of host time got from guest OS"
+          ansible.builtin.set_fact:
+            stat_time: "{{ stat_time_output.stdout_lines[0] }}"
+          when:
+            - stat_time_output.stdout_lines is defined
+            - stat_time_output.stdout_lines | length != 0
+
+        - name: "Display the host time got from guest OS by VMware tools and from ESXi host directly"
+          ansible.builtin.debug:
+            msg:
+              - "In guest OS, VMware tools stat host time is: {{ stat_time }}"
+              - "On ESXi host '{{ esxi_hostname }}', host time is: {{ real_time}}"
+
+        # Calculate how many seconds stat host time is behind comparing with real host time
+        - name: "Time difference between VMware tools stat host time and real host time"
+          ansible.builtin.set_fact:
+            stat_behind_real: "{{ ((real_time | to_datetime) - (stat_time | to_datetime('%d %b %Y %H:%M:%S')) ).total_seconds() }}"
+
+        - name: "Display the time difference"
+          ansible.builtin.debug:
+            msg: "Stat host time in guest OS is behind real host time: {{ stat_behind_real }}s"
+
+        - name: "Check if difference between stat host time and real host time is less than 15s"
+          ansible.builtin.assert:
+            that:
+              - stat_behind_real | float <= 15
+            fail_msg: "Difference between stat host time and real host time is larger than 15s: {{ stat_behind_real }}s"
+            success_msg: "Difference between stat host time and real host time: {{ stat_behind_real }}s"
+      rescue:
+        - include_tasks: ../../common/test_rescue.yml
```

Comment:
The unified name for included tasks test_rescue.yml is "Test case failure" in `docs/TESTCASE_DEV_GUIDE.md`.

Reply:
Updated.